src/database.ts

```diff
@@ -187,6 +207,12 @@ export type CreateSessionResponse = [
 
 export interface CreateSessionOptions {
   labels?: {[k: string]: string} | null;
+  databaseRole?: string | null;
+  gaxOptions?: CallOptions;
+}
+
+export interface GetIamPolicyOptions {
+  requestedPolicyVersion?: number | null;
```

Comment:
This should probably include a comment so customers know what this version does and which one they should be using.